dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -198,3 +205,9 @@ http.proxyredirect.httpsPort.desc=The (secure) port to redirect to. Incoming HTT
 
 http.autoDecompression=Auto decompression
 http.autoDecompression.desc=Specifies whether the HTTP Channel automatically decompresses incoming request body data.
+
+http.compression.types=List of content types for response compression
+http.compression.types.desc=Specifies a list of content types for which the HTTP Channel compresses the response if it is able to do so. Prefix a type with the add (+) character to include the type in addition to the default values. Prefix a type with the remove (-) character to exclude it from compression. Note: The wildcard (*) character is only supported as a content sub-type, such as text/*. 
```

Comment:
**change**
Prefix a type with the add (+) character to include the type in addition to the default values. Prefix a type with the remove (-) character to exclude it from compression. Note: The wildcard (*) character is only supported as a content sub-type, such as text/*.
**to**
To include a content type in addition to the default values, affix the add (+) character as a prefix to that content type. To exclude a content type from compression, affix the remove (-) character as a prefix to that content type. Note: The wildcard (*) character is supported only as a content subtype, such as text/*.